src/types.ts

```diff
@@ -102,7 +102,7 @@ export interface CustomSection extends SectionConfiguration {
 
 export type Section = AutocompleteSection | RecommendationsSection | CustomSection;
 
-export type UserDefinedSection = CustomSection | SectionConfiguration;
+export type UserDefinedSection = Section;
```

Comment:
Why are we changing types on a Storybook PR? I think this is fixed in another PR already so maybe we can skip it here?